src/main/java/com/vdoo/vision/plugin/ScannerAction.java

```diff
@@ -480,14 +480,23 @@ public int getHighlightedIssuesCount() {
     }
 
     public int getHighlightedExposuresCount() {
+        if (highlightedIssues == null) {
```

Comment:
Why not use ternary operators? 😄 
`return highlightedIssues != null ? highlightedIssues.get("exposures").size() : 0`